src/components/street-environment.js

```diff
@@ -19,138 +19,115 @@ AFRAME.registerComponent('street-environment', {
     },
     backgroundColor: { type: 'color', default: '#FFF' }
   },
+
+  init: function () {
+    this.el.sceneEl.renderer.outputColorSpace = THREE.SRGBColorSpace;
+    this.textureLoader = new THREE.TextureLoader();
+    this.light1 =
+      this.el.sceneEl.querySelector('#env-light1') ||
+      this.createLight('env-light1', { type: 'ambient', color: '#FFF' });
+    this.light1.setAttribute('data-layer-name', 'Ambient Light');
+    this.light2 =
+      this.el.sceneEl.querySelector('#env-light2') ||
+      this.createLight('env-light2', { type: 'directional', castShadow: true });
+    this.light2.setAttribute(
+      'data-layer-name',
+      'Directional Light • Shadow Caster'
+    );
+    this.setEnvOption();
```

Comment:
You don't need to call `this.setEnvOption();` in init. It's already called in update. Here it's executed twice when the component initialize.